settlement/bitflyer/upload.go

```diff
@@ -295,139 +285,154 @@ func createBitflyerRequests(
 }
 
 func getSettlementGroup(
-	settlementRequests [][]settlement.Transaction,
+	settlementRequests [][]settlement.AggregateTransaction,
 	toAdd int,
-) ([]settlement.Transaction, int) {
+) ([]settlement.AggregateTransaction, int) {
 	requestSeries := settlementRequests
 	if len(requestSeries) == 0 {
-		set := []settlement.Transaction{}
+		set := []settlement.AggregateTransaction{}
 		return set, 0
 	}
 	lastIndex := len(requestSeries) - 1
 	set := requestSeries[lastIndex]
 	futureLength := len(requestSeries[lastIndex]) + toAdd
 	if futureLength > 1000 {
-		set := []settlement.Transaction{}
+		set := []settlement.AggregateTransaction{}
 		return set, len(settlementRequests) - 1
 	}
 	return set, len(settlementRequests) - 1
 }
 
+// PrepareRequests prepares requests
+func PrepareRequests(
+	ctx context.Context,
+	bitflyerClient bitflyer.Client,
+	txs []settlement.Transaction,
+	excludeLimited bool,
+) (*PreparedTransactions, error) {
+	logger, err := appctx.GetLogger(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	quote, err := bitflyerClient.FetchQuote(ctx, "BAT_JPY", true)
```

Comment:
It's only used twice, so might be overkill for a constant, but I suggest we do it.